coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -63,6 +63,10 @@
         'args': '--wait',
         'gui': True
     },
+    'brackets': {
+        'file_arg_template': '{filename}',
+        'gui': True
```

Comment:
add trailing comma

Reply:
@jayvdb this is the last element. Can you please explain why there should be a trailing comma after `'gui'` mapping?
I saw other editors too there was no trailing comma.

Reply:
Although not syntactically incorrect and hence not absolutely necessary, it is a part of PEP-8. Check the [official style guide](
https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas).

Reply:
@avinashshenoy97 since this type of syntax was not followed for other editors, I think we should keep it like this. Please tell me if I am wrong. @jayvdb

Reply:
You are wrong.